api/src/pcapi/tasks/decorator.py

```diff
@@ -62,15 +70,34 @@ def handle_task(body: payload_type = None):  # type: ignore [no-untyped-def]
 
         try:
             f(body)
+
+        except requests.ExternalAPIException as exception:
+            if not exception.is_retryable:
+                # The task should not be retried as it would result with the same error again.
+                # A log with a hiher level should happen before.
+                logger.warning("Task %s failed and should not be retried", path, extra=job_details)
+                return
+
+            if retry_attempt and int(retry_attempt) + 1 >= settings.CLOUD_TASK_MAX_ATTEMPTS:
+                # notify that External API must be anavailable
+                logger.error(  # pylint: disable=logging-fstring-interpolation
+                    f"External API unavailable for CloudTask {path}",
+                    extra=job_details | {"exception": str(exception), "cause_exception": str(exception.__cause__)},
+                )
+            else:
+                logger.warning(
+                    "The cloud task has not been executed successfully: %s",
```

Comment:
(détail) Pour aider le développeur, je propose d'être plus explicite sur le fait qu'il n'y a rien à faire puisque la tâche va être à nouveau tentée : 

```suggestion
                    "The cloud task has failed and will automatically be retried: %s",
```